py3status/modules/dnf.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+"""
+Displays the number of package updates pending for a Fedora Linux installation.
+This will display a count of how many `dnf` updates are waiting
+to be installed.
+Additionally check if any update security notices.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh this module in seconds
+      default is 600
+    - format : display format
+      default is 'DNF: {updates}'
+    - color_good : color when no upgrades needed
+      default is None
```

Comment:
I'd suggest changing "default is None" to "defaults to global color_good" and so on for the other two colors.